project_scrum/project_scrum_view.xml

```diff
@@ -0,0 +1,1121 @@
+							<!--User Stories -->
+							<page string="User Story">
+								<field colspan="2" col="1" name="us_ids"
+									context="{
+                                        'default_project_id': project_id,
+                                        'default_sprint_id': id,
+                                        }">
+									<tree>
+										<field name="name" />
+										<field name="actor_ids" />
+										<field name="sprint_ids" />
+										<field name="project_id" />
+									</tree>
+								</field>
+							</page>
+							<page string="Review">
+								<!--page for review -->
+								<group col="1">
+									<field name="review" />
+								</group>
+								<group col="1">
+									<field name="task_ids" string="Task"
+										context="{
+                                            'default_project_id': project_id,
+                                            'default_sprint_id': id,
+                                            'default_us_ids': us_ids,
+                                            }">
+										<tree>
+											<field name="sequence" widget="handle" />
+											<field name="priority" />
+											<field name="name" />
+											<field name="user_id" />
+											<field name="date_deadline" />
+											<field name="stage_id" />
+										</tree>
+									</field>
+								</group>
+							</page>
+							<page string="Retrospective">
+								<!--page for retropective -->
+								<field name="retrospective" />
+							</page>
+						</notebook>
+					</sheet>
+				</form>
+			</field>
+		</record>
+
+		<!--sprint calendar view -->
+		<record id="view_ps_sprint_calendar" model="ir.ui.view">
+			<field name="name">project.scrum.sprint.calendar</field>
+			<field name="model">project.scrum.sprint</field>
+			<field name="type">calendar</field>
+			<field name="arch" type="xml">
+				<calendar string="Scrum Sprint" date_start="date_start"
+					color="us_ids">
+					<field name="name" />
+					<field name="project_id" />
+				</calendar>
+			</field>
+		</record>
+
+		<!--sprint gantt view -->
+		<record id="view_ps_sprint_gantt" model="ir.ui.view">
+			<field name="name">project.scrum.sprint.gantt</field>
+			<field name="model">project.scrum.sprint</field>
+			<field name="type">gantt</field>
+			<field name="arch" type="xml">
+				<gantt date_start="date_start" date_stop="date_stop" progress="progress"
+					string="Projects">
+				</gantt>
+			</field>
+		</record>
+
+		<!--sprint graph view -->
+		<record id="view_ps_sprint_graph" model="ir.ui.view">
+			<field name="name">project.scrum.sprint.graph</field>
+			<field name="model">project.scrum.sprint</field>
+			<field name="type">graph</field>
+			<field name="arch" type="xml">
+				<graph string="Scrum Sprint" type="bar">
+					<field name="project_id" type="row" />
+				</graph>
+			</field>
+		</record>
+
+		<!--Scrum User Stories -->
+		<!--user stories kanban view -->
+		<record id="view_ps_us_kanban" model="ir.ui.view">
```

Comment:
your indentation is too long and makes the code difficult to read for maintenance/review. could you switch to 4 spaces or less?